Dockerfile-nsv

```diff
@@ -15,6 +15,9 @@ RUN apt install -y cmake
 # Install librdkafka.
 RUN apt-get install -y libsasl2-dev libsasl2-modules libssl-dev librdkafka-dev
 
+# install python for testing
```

Comment:
With the addition of Dockerfile.testing should we drop the python install from this nsv and the standalone dockerfiles?

Reply:
Yup good call! I've removed the python installation from these dockerfiles.